### PR TITLE
ddl: GetDDLJobs should return a sort job.

### PR DESCRIPTION
**Pull request by crazycs520**
## What have you changed? (mandatory)
Function `GetDDLJobs` adds sort jobs before return.

## How has this PR been tested? (mandatory)
No

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)
No

## Does this PR affect tidb-ansible update? (mandatory)
No

## Does this PR need to be added to the release notes? (mandatory)
No

### CODE REVIEW

**Comment by crazycs520** [comment]
@winkyao @zimulala @ciscoxll  PTAL

File: util/admin/admin.go [hunk 1]
```diff
@@ -147,7 +148,23 @@ func GetDDLJobs(txn kv.Transaction) ([]*model.Job, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	return append(generalJobs, addIdxJobs...), nil
+	jobs := append(generalJobs, addIdxJobs...)
```
Comment (winkyao) [inline util/admin/admin.go]: Remove line 140.

File: util/admin/admin_test.go [hunk 2]
```diff
@@ -121,6 +121,57 @@ func (s *testSuite) TestGetDDLJobs(c *C) {
...
+	t := meta.NewMeta(txn)
+	enQueueDDLJobs(c, t, model.ActionCreateTable, 0, 5)
+
+	// insert add index jobs to AddIndexJobListKey queue
+	t = meta.NewMeta(txn, meta.AddIndexJobListKey)
+	enQueueDDLJobs(c, t, model.ActionAddIndex, 5, 10)
+
+	// insert 5 drop table jobs to DefaultJobListKey queue
+	t = meta.NewMeta(txn)
+	enQueueDDLJobs(c, t, model.ActionDropTable, 10, 15)
```
Comment (winkyao) [inline util/admin/admin_test.go]: You enQueue the job in order. Why not enQueue id 10-15 jobs first?

File: util/admin/admin.go (hunk 1 above)
Comment (zimulala) [reply util/admin/admin.go]: Please address this comment.

File: util/admin/admin.go [hunk 3]
```diff
@@ -135,8 +136,7 @@ func getDDLJobsInQueue(t *meta.Meta, jobListKey meta.JobListKeyType) ([]*model.J
 	return jobs, nil
 }
 
-// GetDDLJobs returns all DDL jobs.
-// TODO: Sort jobs.
+// GetDDLJobs get all DDL jobs and sorts jobs by job.ID before return.
```
Comment (shenli) [inline util/admin/admin.go]: What's mean by  "before return"?

File: util/admin/admin_test.go [hunk 4]
```diff
@@ -121,6 +121,56 @@ func (s *testSuite) TestGetDDLJobs(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func checkJobsSort(jobs []*model.Job) bool {
```
Comment (shenli) [inline util/admin/admin_test.go]: s/checkJobsSort/isJobsSorted/

**Pull request merged**